src/School39.Landing/Components/ContactusComponent.razor

```diff
@@ -1,82 +1,61 @@
 <section id="contact" class="contact">
```

Comment:
Aka file ni nomiga qayerdan review ochishni bilmadim, shuning uchun 1 qatorga ochidim. File nomida sal notog'ri bolib qobti shekilli. Sizda ContactUsComponent.razor ekan, adashmasam ContactsComponent.razor bolishi kerak edi yaniy **_u_** ortiqcha emasmi?

Reply:
Ha shunday to'g'irlab yozaman